### PR TITLE
Craft Shop

### PR DESCRIPTION
**Pull request by nisargshh**
As mentioned in #7779 add the terms hobby to craft shop

### CODE REVIEW

File: data/presets/presets/shop/craft.json [hunk 1]
```diff
@@ -1,16 +1,9 @@
 {
     "icon": "fas-cut",
-    "geometry": [
-        "point",
-        "area"
-    ],
+    "geometry": ["point", "area"],
```
Comment (quincylvania) [inline data/presets/presets/shop/craft.json]: It's not a big deal but please leave arrays on multiple lines if that's how they're already formatted.

Comment (nisargshh) [reply data/presets/presets/shop/craft.json]: My bad I thought the lint would fix that.

Comment (quincylvania) [reply data/presets/presets/shop/craft.json]: We don't autoformat the preset JSON at the moment…

**Comment by nisargshh** [comment]
Commited requested changes

**Comment by quincylvania** [comment]
@nisargshh Thanks!

**Pull request merged**